docs/devices/ZBDongle-E.md

```diff
@@ -23,12 +23,27 @@ pageClass: device-page
 
 
 <!-- Notes BEGIN: You can edit here. Add "## Notes" headline if not already present. -->
-
+## Notes
+
+### How to Create a SONOFF ZBDongle-E Router
+You can create a powerful ZigBee router from an ordinary Sonoff Zigbee 3.0 USB Dongle Plus (Model "ZBDongle-E" with EFR32MG21 chip) by flashing a router firmware onto the device. You can find the official guide for flashing the device here: [SONOFF Zigbee 3.0 USB dongle plus firmware flashing](https://sonoff.tech/wp-content/uploads/2022/11/SONOFF-Zigbee-3.0-USB-dongle-plus-firmware-flashing-.pdf). The guide also contains the link to the official [SONOFF Router Firmware for the ZBDongle-E](https://github.com/itead/Sonoff_Zigbee_Dongle_Firmware/tree/master/Dongle-E/Router).
+### Hints
+The guide from SONOFF is not very detailed. Here are some additional hints:
+* First, you have to disassemble the device and pull the logic board out of its housing. You need a **J00 Phillips screwdriver** for this.
+  <img src="https://github.com/TomTonic/zigbee2mqtt.io/blob/master/docs/devices/dongle-e-screws.jpg" width="130" height="129"/>
+  <img src="https://github.com/TomTonic/zigbee2mqtt.io/blob/master/docs/devices/dongle-e-board-in-housing.jpg" width="130" height="117"/>
```

Comment:
Please add the picture directly to the repo.

Reply:
I added the pictures to this patch. Did I manage to hit the right location for them?

Reply:
perfect!